plugins/search-backend/src/service/router.ts

```diff
@@ -46,8 +72,8 @@ export async function createRouter(
       );
 
       try {
-        const results = await engine?.query(req.query);
-        res.send(results);
+        const resultSet = await engine?.query(req.query);
```

Comment:
Have you considered filtering when documents are collected and indexed? Is there a better reason to do this here? For understanding only 🙂

Reply:
The assumption here is that the collator is naive and a malicious user can induce the collator to write a location which is unsafe or similarly that the DB has been compromised. This is reduces the impact of this by not trusting what's written to the DB.

Reply:
Got it, thanks for explaning!